### ISSUE TITLE
Visual glitch in Data Table component under some (unknown) conditions.

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#678]
@mlinnem commented on [Wed Mar 28 2018](https://github.com/carbon-design-system/carbon-components-react/issues/758)

## Detailed description

> Describe in detail the issue you're having.

There is a graphical glitch in a specific instance of the data table as it is currently being used in the Container Service Image Report Detail" page of IBM Cloud.

In the collapsed state, it appears as a the bottom of the table row being elevated by one pixel for the first 30 or so horizontal pixels of the row.

<img width="119" alt="screen shot 2018-03-28 at 10 51 35 am" src="https://user-images.githubusercontent.com/1547834/38042922-4636de08-327b-11e8-8bed-a639b4d7042e.png">

In the expanded state, it appears as a white line mysteriously floating in the row.

<img width="177" alt="screen shot 2018-03-28 at 10 51 49 am" src="https://user-images.githubusercontent.com/1547834/38042945-5658a690-327b-11e8-85e1-cb17a3f51156.png">

The same glitch does not appear on the a very similar table on the same page.

<img width="441" alt="screen shot 2018-03-28 at 11 30 08 am" src="https://user-images.githubusercontent.com/1547834/38043013-799375fe-327b-11e8-9a82-c9aee6e44337.png">

> Is this a feature request (new component, new icon), a bug, or a general issue?

A bug.

> Is this issue related to a specific component?

Related to data table.
> What browser are you working in?

Version 65.0.3325.181 (Official Build) (64-bit)

Does not occur in Safari (Version 11.0.3 (13604.5.6))

> What version of the Carbon Design System are you using?

Unsure.

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

In prod right now of Container Service.

---

@joshblack commented on [Wed Mar 28 2018](https://github.com/carbon-design-system/carbon-components-react/issues/758#issuecomment-376964306)

Hey @mlinnem! 👋 Thanks so much for taking the time to make this issue 😄 

Is this related to the styles of this component? If so, I can move it over to `carbon-components` so it gets addressed there 🙂

---

@mlinnem commented on [Wed Mar 28 2018](https://github.com/carbon-design-system/carbon-components-react/issues/758#issuecomment-377008447)

@joshblack Yep, thank you for looking at it! :)

It may be. I see now that the glitch disappears when I remove the css rule "-webkit-box-sizing: border box". I'm just a designer and not a web science man, but that would seem to mean the cause is styling? A bunch of other stuff goes wrong when I disable that selector... (e.g. expand/collapse arrows on tables get bigger).

---

@joshblack commented on [Thu Mar 29 2018](https://github.com/carbon-design-system/carbon-components-react/issues/758#issuecomment-377275258)

Thanks for diving into this! Definitely seems like a style issue so will move it over there. Thank you again for your time and effort in identifying this bug, we really appreciate it! 🎉

**Comment by joshblack** [comment]
cc @mlinnem is there any chance to get a reproducible case for this issue? Would help out a ton with figuring out what exactly is going on here.

**Comment by mlinnem** [comment]
So I just tried to reproduce it, and could only get it to manifest if I set my zoom to 110% (that is, by pressing command-plus in chrome). 

I don't know if this used to happen under normal zoom conditions, or when I reported it if I had my zoom set to 100%, but in any case it's not there any more for the normal case. I imagine it is not worthwhile to fix if it only appears when zoomed, but in case you are interested, I can get it to occur on carbondesignsystem.com still.

<img width="121" alt="screen shot 2018-05-18 at 2 50 03 pm" src="https://user-images.githubusercontent.com/1547834/40254887-cce82496-5aaa-11e8-8339-5cbcc39bd882.png">
